packages/rocketchat-api/server/v1/groups.js

```diff
@@ -1,10 +1,15 @@
 //Returns the private group subscription IF found otherwise it will reutrn the failure of why it didn't. Check the `statusCode` property
-function findPrivateGroupById({ roomId, userId, checkedArchived = true }) {
-	if (!roomId || !roomId.trim()) {
+function findPrivateGroupByIdOrName({ roomId, roomName, userId, checkedArchived = true }) {
+	if ((!roomId || !roomId.trim()) && (!roomName || !roomName.trim())) {
 		return RocketChat.API.v1.failure('Body param "roomId" is required');
```

Comment:
Thanks for making the changes, however this message now needs to be updated since either `roomId` or `roomName` are required.